tests/plugins/tableselection/mouseover.js

```diff
@@ -0,0 +1,43 @@
+/* bender-tags: tableselection */
+/* bender-ckeditor-plugins: tableselection */
+/* bender-include: _helpers/tableselection.js */
+
+( function() {
+	'use strict';
+
+	bender.editors = {
+		scrollable: {
```

Comment:
Please rename this property to `classic` and add a `divarea` editor too, see https://github.com/ckeditor/ckeditor-dev/blob/b8d9a84/tests/core/editable/readonly.js#L36-L41